crates/compiler/roar/src/proc.rs

```diff
@@ -0,0 +1,30 @@
+use crate::ops::{Operation};
+use crate::storage::{Register,FloatRegister,Constant,ByteSize,Offset,Input,Output};
+use roc_module::symbol;
+#[derive(Clone,Debug)]
+#[non_exhaustive]
+pub struct Proc {
+    ///The instructions themselves
+    pub instructions : Vec<Operation>,
```

Comment:
These should be a bump slice if they will stay one length after allocation. If they can change length, probably should be a bumpalo::Vec. Preferably though we avoid changing length if possible. Ditto with inputs below (though inputs is almost certainly just a slice)